app/konnect/org-management/sso.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Configure Generic SSO
+---
+
+
+As an alternative to {{site.konnect_saas}}’s native authentication, you can set up single sign-on (SSO) access to {{site.konnect_short_name}} using OpenID Connect or SAML. This authentication method allows your users to log in to {{site.konnect_saas}} using their IdP credentials, without needing a separate login. This topic covers configuring SSO for use with various identity providers.
+
+We currently support the following providers: 
+* Okta 
+* Azure Active Directory
+* Oracle Identity Cloud Service 
+* Keycloak. 
+
+If you want to configure Okta, please see the [Okta configuration guide](/konnect/org-management/sso/).
+
+## Prerequisites
+
+* {{site.konnect_short_name}} must be added to your IdP as an application
+* Claims are set up in your IdP
+
+
+## Set up SSO in {{site.konnect_short_name}}
+{% navtabs %}
+{% navtab OIDC%}
+1. In [{{site.konnect_saas}}](https://cloud.konghq.com/login), click {% konnect_icon organizations %} **Organization**, and then **Auth Settings**.
+
+1. Click **Configure provider** for **OIDC**.
+
+1. Paste the issuer URI from your IdP in the **Issuer URI** box. 
+
+1. Paste the client ID from your IdP in the **Client ID** box.
+
+1. Paste the client secret from your IdP in the **Client Secret** box.
+
+1. In the **Organization Login Path** box, enter a unique string. For example: `examplepath`.
+
+    {{site.konnect_short_name}} uses this string to generate a custom login
+    URL for your organization.
+
+    Requirements:
+    * The path must be unique *across all {{site.konnect_short_name}} organizations*.
+    If your desired path is already taken, you must to choose another one.
+    * The path can be any alphanumeric string.
+    * The path does not require a slash (`/`).
+
+1. After clicking Save, configure the SP Entity ID and Login URL on your SAML IdP.
```

Comment:
Let's not forget to mention the OIDC provider needs to be enabled. Something along those lines?
```suggestion
1. After clicking Save, close the configuration dialog and click Enable on your OIDC provider.
```